src/metrics/prometheus/runner.go

```diff
@@ -101,6 +101,9 @@ func (c *PrometheusRunner) Run(ctx context.Context) error {
 				geoDistribution.WithLabelValues(k).Set(v)
 			}
 
+			// Temporary solution to test the DB feature
+			c.PeerStore.PersistToDb()
```

Comment:
Could you write some unit tests for the function? Perhaps you could get some inspiration by [this](https://github.com/migalabs/armiarma/blob/0b31aaef238de87995d4284a406d3e1438970574/src/metrics/peerstore_test.go) one I wrote. Perhaps you can add a new function in the same file `peerstore_test.go` with a `Test_PeerStorePersistToDb` or something like that, where you create a "fake" peer store, populate it with some parameters and then call `PersistToDb`. Then you can open the db and assert that the content is the expected one :)